app/src/main/res/layout-sw600dp-land/onboarding_slide.xml

```diff
@@ -20,10 +20,10 @@
       android:layout_height="0dp"
       android:adjustViewBounds="true"
       android:contentDescription="@{viewModel.contentDescription}"
-      android:scaleType="fitXY"
+      android:scaleType="centerCrop"
       android:src="@{viewModel.slideImage}"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintDimensionRatio="H, 10:9"
+      app:layout_constraintEnd_toStartOf="@+id/guideline3"
```

Comment:
Ok, I'll give it a try

Reply:
Done!  
PTAL  
I made these changes:  
-Used `app:layout_constraintDimensionRatio="H, 10:9"` and `android:scaleType="fitXY"` instead of `guideline` and `android:scaleType="centerCrop"`
-Centered the `textViews` between the `imageView` and the end of the `parentLayout` and gave them a fixed width of `360dp` instead of placing the `textViews` relative to the end of the `parentLayout` only  
Please review them and tell me if there is something that needs to change

Reply:
We should avoid giving fixed widths. So mainly you will need to shift to relative positioning which is `96dp` as mentioned in the email.

Also, please convert this PR to final PR and then assign it back to me. Ideally a PR should be assigned as final PR for full review so that I can leave comments accordingly.